cernopendata/modules/fixtures/data/docs/cod-portal-upgrade-31-1-2024/cod-portal-upgrade-31-1-2024.md

```diff
@@ -0,0 +1,14 @@
+A new version of the CERN Open Data Repository has been deployed in January 2024.
+Some of the highlights of this release are:
+
+* Change of technologies, from Angular to [ReactJS](https://react.dev/) and from Bootstrap to [Semantic UI](https://semantic-ui.com/)
+for the web interface, which improves the navigation and overall use experience.
+* Changes in the search functionality, including modifying the default search to do an `AND` of the terms (instead of `OR`),
```

Comment:
What about Improvements instead of Changes to highlight the _positive_ aspects of changes?

Reply:
Done